tests/sample_app/README.md

```diff
@@ -0,0 +1,26 @@
+# CakePHP Application Skeleton
```

Comment:
Am still customizing the sample_app that is used to showcase tests and at the same time act as our integration tests.